packages/core/package.json

```diff
@@ -233,6 +233,85 @@
                     "type": "object",
                     "description": "%AWS.configuration.description.lambda.recentlyUploaded%",
                     "default": []
+                },
+                "aws.amazonq.logLevel": {
+                    "type": "string",
+                    "default": "debug",
+                    "enum": [
+                        "error",
+                        "warn",
+                        "info",
+                        "verbose",
+                        "debug"
```

Comment:
side-note: we won't need a "loglevel" setting in the future, we can use the OutputChannel to control that instead.

<img width="560" alt="image" src="https://github.com/aws/aws-toolkit-vscode/assets/55561878/122962e6-c6d5-459b-ac10-25fb53ab6c28">